docs/resources/sql_table.md

```diff
@@ -0,0 +1,97 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_sql_table (Resource)
+
+Within a metastore, Unity Catalog provides a 3-level namespace for organizing data: Catalogs, databases (also called schemas), and tables / views.
+
+A `databricks_sql_table` is contained within [databricks_schema](schema.md).
```

Comment:
```suggestion
A `databricks_sql_table` is contained within [databricks_schema](schema.md), and can represent either a managed table, an external table or a view.

This resource creates and updates the Unity Catalog table/view by executing the necessary SQL queries on a special auto-terminating cluster it would create for this operation.
```